### DISCUSSION TITLE
HMS-1000: Blink-Code grün alle 4 sec

### DISCUSSION THREAD

**Discussion by Fokker-1** [tbnobody/OpenDTU#2165] [Q&A]
Hallo,

mein HMS-1000 blinkt abends gelegentlich alle 4 Sekunden grün.
Ich habe mir nun schon einige Anleitungen geladen, aber in keiner ist dieser Blinktyp gennant.

Hat jemand dazu eine Information?

Schon mal Danke im Voraus!

Gruß
Werner

**Comment by ghost** [comment]
Hi,
klar doch.
 
"Langsames grünes Blinken (4 s Abstand): Strom wird erzeugt, jedoch keine Kommunikation mit DTU."

Bedeutet alles OK. (Strom wird erzeugt, keine DTU verbunden)

Quelle:
[Handbuch](https://www.hoymiles.com/wp-content/uploads/downloadupload/User%20Manual_HM-1000-1200-1500_Global_DE_V202206.pdf)

**Reply by Fokker-1** [reply]
Hallo,

danke für die Info, das hilft weiter.

Aufgrund der aktuellen Situation bei Auftreten dieses Blink-Codes (OpenDTU ist dabei orange) hatte ich den im Handbuch beschrieben Status dazu auch  schon vermutetet.

Der Link geht zu einem Handbuch für die HM-Typen, da steht diese Angabe drin.

Ich habe einen HMS-Typ und da habe ich - wie oben schon genannt - bisher keine entsprechende Angabe zu diesem Blink-Code gefunden.

Daher habe ich auch mal den Hoymiles-Service angeschrieben - und warte mal ...

Noch mal danke und Grüße
Werner

**Comment by Fokker-1** [comment]
Hallo,

ich komme heir noch mal auf das Theme mit "langsames grünes Blinken alle 4 Sek." zurück.

Ich habe mir das jetzt noch mal abends genauer angesehen.

Diese o. g. Info zum Blinken stammt aus einer HM-BA:
"Langsames grünes Blinken (4 s Abstand): Strom wird erzeugt, jedoch keine Kommunikation mit DTU."

Bei mir tritt das immer dann auf, nachdem der Status von "Blau" nach "orange (gelb?) gewechselt hat und bleibt über die gesamte Zeit so stehen, in der dieser Orange-tatus besteht.

"Orange" bei der DTU bedeutet m. W., dass **KEIN** Strom produziert wird, aber dass die DTU mit dem WR weiterhin kommunizieren kann - oder habe ich dass falsch verstanden?

Die obige Definition "Langsames grünes Blinken (4 s Abstand): Strom wird erzeugt, jedoch keine Kommunikation mit DTU." habe ich in einer HM-BA mit der Version "Global 202203" für HM-Typen gefunden, allerdings finde ich die in neueren HM-BA-Version nicht mehr und auch in keiner HMS-BA-Version.

Ich hatte den Service von Hoymiles kontaktiert, dort kannte man wohl dieses Verhalten (4x grün langsam Blinken) beim MMS-1000 nicht, ganz eigentümlich. Ich habe nun auf Wunsch noch weitere Infos dort hin geschickt, mal sehen, ob da noch Erhellendes kommt ...

Gruß
Werner
